src/applications/disability-benefits/all-claims/tests/pages/physicalHealthChanges.unit.spec.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { DefinitionTester } from '../../../../../platform/testing/unit/schemaform-utils.jsx';
+import { mount } from 'enzyme';
+import formConfig from '../../config/form';
+
+describe('Physical Changes 781a', () => {
+  const {
+    schema,
+    uiSchema,
+  } = formConfig.chapters.disabilities.pages.physicalHealthChanges;
+
+  it('should render', () => {
+    const form = mount(
+      <DefinitionTester
+        definitions={formConfig}
+        schema={schema}
+        uiSchema={uiSchema}
+        data={{
+          physicalChanges: {
+            'view:other': true,
+          },
+        }}
+        formData={{}}
+      />,
+    );
+
+    expect(form.find('input').length).to.equal(8);
+  });
+
+  it('should not submit if no options selected', () => {
```

Comment:
Change this to `should submit if no options selected`

Reply:
Fixed!